### ISSUE TITLE
Genome alignment visualization causes error when using gz compressed files

### ISSUE THREAD

**Issue by erinyoung** [moshi4/pyGenomeViz#64]
I was updating pyGenomeViz and going through the examples on your README.

I was trying to follow
```
# Download example dataset
pgv-download escherichia_coli
# Run progressiveMauve CLI workflow
pgv-pmauve NC_000913.gbk.gz NC_002695.gbk.gz NC_011751.gbk.gz NC_011750.gbk.gz \
           -o pgv-pmauve_example --show_scale_bar
```

And this is the output and error that I observed. Do you know if I may be missing a dependency somewhere?
```
61.89 2024-09-25 18:07:04 | INFO | $ pgv-pmauve NC_000913.gbk.gz NC_002695.gbk.gz NC_011751.gbk.gz NC_011750.gbk.gz -o pgv-pmauve_example --show_scale_bar
61.89 2024-09-25 18:07:04 | INFO | Operating System: linux
61.89 2024-09-25 18:07:04 | INFO | Python Version: v3.9.17
61.89 2024-09-25 18:07:04 | INFO | Check Dependencies: matplotlib v3.9.2
61.89 2024-09-25 18:07:04 | INFO | Check Dependencies: biopython v1.84
61.89 2024-09-25 18:07:04 | INFO | Parameters: seqs=['NC_000913.gbk.gz', 'NC_002695.gbk.gz', 'NC_011751.gbk.gz', 'NC_011750.gbk.gz']
61.89 2024-09-25 18:07:04 | INFO | Parameters: outdir='pgv-pmauve_example'
61.89 2024-09-25 18:07:04 | INFO | Parameters: formats=('png', 'html')
61.89 2024-09-25 18:07:04 | INFO | Parameters: reuse=False
61.89 2024-09-25 18:07:04 | INFO | Parameters: quiet=False
61.89 2024-09-25 18:07:04 | INFO | Parameters: debug=False
61.89 2024-09-25 18:07:04 | INFO | Parameters: fig_width=15
61.89 2024-09-25 18:07:04 | INFO | Parameters: fig_track_height=1.0
61.89 2024-09-25 18:07:04 | INFO | Parameters: track_align_type='center'
61.89 2024-09-25 18:07:04 | INFO | Parameters: feature_track_ratio=0.25
61.89 2024-09-25 18:07:04 | INFO | Parameters: show_scale_bar=True
61.89 2024-09-25 18:07:04 | INFO | Parameters: show_scale_xticks=False
61.89 2024-09-25 18:07:04 | INFO | Parameters: curve=False
61.89 2024-09-25 18:07:04 | INFO | Parameters: dpi=300
61.89 2024-09-25 18:07:04 | INFO | Parameters: track_labelsize=20
61.89 2024-09-25 18:07:04 | INFO | Parameters: scale_labelsize=15
61.89 2024-09-25 18:07:04 | INFO | Parameters: normal_link_color='grey'
61.89 2024-09-25 18:07:04 | INFO | Parameters: inverted_link_color='red'
61.89 2024-09-25 18:07:04 | INFO | Parameters: refid=0
61.89 2024-09-25 18:07:04 | INFO | Parameters: block_plotstyle='box'
61.89 2024-09-25 18:07:04 | INFO | Parameters: block_cmap='gist_rainbow'
61.89 2024-09-25 18:07:04 | ERROR | 'NC_000913.gbk.gz' is invalid file suffix (valid_suffixes=('.gb', '.gbk', '.gbff', '.gb.gz', '.gbk.gz', '.gbff.gz', '.fa', '.fna', '.fasta', '.fa.gz', '.fna.gz', '.fasta.gz'))
```

**Comment by moshi4** [comment]
Hi @erinyoung,

Thank you for the bug report. The check code for the gz compressed file extension was incorrect and has been fixed in latest [v1.4.1](https://github.com/moshi4/pyGenomeViz/releases/tag/v1.4.1) release.

**Comment by erinyoung** [comment]
Thank you!
